components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1437,6 +1437,9 @@
 <statefreq> 480 </statefreq>
 <integration> "explicit" </integration>
 
+<cubed_sphere_map>0</cubed_sphere_map>
```

Comment:
I don't think you want to set a default here. The default behavior in HOMME is to have this set at runtime, but setting an explicit value for a default will override that. The following lines in `homme/src/share/namelist_mod.F90` are relevant here:
```
    if (cubed_sphere_map<0) then
#if ( defined MODEL_THETA_C || defined MODEL_THETA_L ) 
       cubed_sphere_map=2  ! theta model default = element local
#else  
       cubed_sphere_map=0  ! default is equi-angle gnomonic
#endif 
       if (ne.eq.0) cubed_sphere_map=2  ! must use element_local for var-res grids
#ifdef CAM
       if (fv_nphys.gt.0) cubed_sphere_map=2  ! must use element_local for FV physics grid
#endif
    endif
```
So I think we want to leave the default unset (the code sets `cubed_sphere_map = -1`), and if a value is not provided via namelist then the code determines the proper defaults. Note that this may or may not be the "ideal" behavior for reading this value in, but I think to work with how it reads the value in now we should leave `cubed_sphere_map` unset, or give it a value of `-1` here. @mt5555 or @oksanaguba could you comment here?

Reply:
A default needs to be set, else the build-namelist will crash if we try to run a grid other than ne1024np4.  If a default should not be specified for this variable, then I think the proper place to set the namelist value should be placed in the components/cam/bld/namelist_files/use_cases/2000_scream_hr_av1c-04p2.xml file, right?

Reply:
In that case, I think a default of -1 should be set here, which should force the code to determine the default (which is the current behavior). That should prevent breaking current configurations I think. I don't think we want it in the use case file, because setting `cubed_sphere_map = 2` is required for any ne1024 configuration (SCREAM or otherwise), because that is the mapping we used for all of the ne1024 (and ne512) input files.

Reply:
cool, i'll set it like that and do a short test with ne30 to make sure that it is still b4b

Reply:
the test was fine.  So i committed and pushed this revision